### PR TITLE
Implement Client Library and Version

### PR DESCRIPTION
**Pull request by Wilsonl9**
Adding the ability for react SDK to specify `_client_library` and `_client_version` for proper analytics tracking.

### CODE REVIEW

**Review by mewelling — COMMENTED** [review]
I feel like dropping browswrap would be a major change, and not sure we want to do that work right now. 

Does the version stuff require any process changes on our part? or do we still just prep all this with a `npm release` ?

**Comment by Wilsonl9** [comment]
We are dropping browser wrap from our demo code, this does not affect the snippet

File: src/PSSnippet.js [hunk 1]
```diff
@@ -93,10 +93,10 @@ function injectSnippet(scriptURL, backupScriptURL) {
 function isSnippetLoaded(psScriptURL) {
 	const scripts = document.getElementsByTagName('script');
 	if (window._ps && window._ps.loaded && window._ps.realThang === 317) return true;
-    for (let i = 0; i < scripts.length; i += 1) {
-      if (scripts[i].src.indexOf(psScriptURL) !== -1){
-		return true;
-	  } 
+	for (let i = 0; i < scripts.length; i += 1) {
+		if (scripts[i].src.indexOf(psScriptURL) !== -1) {
```
Comment (kylepeeler) [inline src/PSSnippet.js:97]: I think we want to null check src because of #39

Comment (mewelling) [reply src/PSSnippet.js:97]: good catch dude ;) 

@Wilsonl9 might as well fix things when we come across them!

**Review by mewelling — APPROVED** [review]

**Pull request merged**